### ISSUE TITLE
Enable es6.mixedImports

### ISSUE THREAD

**Issue by dev101** [dougmoscrop/serverless-plugin-include-dependencies#91]
Any chance to add `es6.mixedImports` option in the [`precinct.paperwork`](https://github.com/dougmoscrop/serverless-plugin-include-dependencies/blob/master/get-dependency-list.js#L80) call?

I have to use some ES modules via dynamic imports like `const aModule = (await import('aModule')).default` , and these are not detected by the default precinct config, thus not included in the package.

**Comment by dougmoscrop** [comment]
yeah seems reasonable..  though to be honest, I don't know why people don't just use esbuild at this point.

**Comment by dev101** [comment]
@dougmoscrop you mean https://github.com/floydspace/serverless-esbuild ?
never heard of it until today.

serverless-plugin-include-dependencies worked just fine for me as I primarily maintain legacy cjs code, until some day some dependencies are only available as es modules.

nevertheless, on your side it's just about adding an option to `paperwork` and upgrading `precint`, seems to be backwards-compatible and no breaking changes, huh?
